modules/nw-sriov-configuring-operator.adoc

```diff
@@ -11,19 +11,46 @@ SriovOperatorConfig Custom Resource (CR).
 
 [NOTE]
 =====
-There is one Custom Resource of SriovOperatorConfig being created when the
-operator is up. Change the operator configuration by modifing this pre-created
-Custom Resource.
+There is one Custom Resource of SriovOperatorConfig being created when
```

Comment:
Why are you changing this file?

Reply:
> Why are you changing this file?

These aren't my changes. They must have come from the bad rebase.